lint.sh

```diff
@@ -38,23 +14,8 @@ if [ $? -ne 0 ]; then
   echo '$ pip install $(cat dev/lint-requirements.txt | grep "black==") && black .'
 fi
 
-echo -e "\n========== pycodestyle ==========\n"
-exclude=$(join "," "${exclude_dirs[@]}")
-include=$(join " " "${include_dirs[@]}")
-pycodestyle --max-line-length=100 --ignore=E203,W503 --exclude=$exclude -- $include
-
 echo -e "\n========== pylint ==========\n"
-# pylint's `--ignore` option filters files based on their base names, not paths.
-# see: http://pylint.pycqa.org/en/latest/user_guide/run.html#command-line-options
-# This behavior might cause us to unintentionally ignore some files.
-# To avoid this issue, select files to lint using `git ls-files` and `grep`.
-# Another advantage of this approach is we can apply pylint to all python scripts
-# without creating `__init__.py` in all directories.
-exclude="^\($(join "\|" "${exclude_dirs[@]}")\)/.\+\.py$"
-include="^\($(join "\|" "${include_dirs[@]}")\)/.\+\.py$"
-msg_template="{path} ({line},{column}): [{msg_id} {symbol}] {msg}"
-
-pylint --jobs=0 --msg-template="$msg_template" --rcfile="$FWDIR/pylintrc" $(git ls-files | grep $include | grep -v $exclude)
```

Comment:
I made the following changes:

- Move `--jobs` is specified in `pylintrc`.
- Removed `--msg-template` since the default template is good enough.
- Removed `--rcfile` because `pylint` automatically detects `pylintrc`.